### ISSUE TITLE
Second a_assay is missing instrument

### ISSUE THREAD

**Issue by sneumann** [ISA-tools/mzml2isa#27]
Hi, I have a study with pos and neg mzML files, and the second assay seems to be missing 
the Platform:
![image](https://cloud.githubusercontent.com/assets/195385/24548846/eedd2e68-1616-11e7-97a5-d2fec12a0bb8.png)

and indeed, `Study Assay Technology Platform` is only written out for the first assay:

```
STUDY ASSAYS
Study Assay File Name	"a_MTBLSX_POS_metabolite_profiling_mass_spectrometry.txt"	"a_MTBLSX_NEG_metabolite_profiling_mass_spectrometry.txt"
Study Assay Measurement Type	"metabolite profiling"	"metabolite profiling"
Study Assay Measurement Type Term Accession Number	"http://purl.obolibrary.org/obo/OBI_0000366"	"http://purl.obolibrary.org/obo/OBI_0000366"
Study Assay Measurement Type Term Source REF	"OBI"	"OBI"
Study Assay Technology Type	"mass spectrometry"	"mass spectrometry"
Study Assay Technology Type Term Accession Number	"http://purl.obolibrary.org/obo/OBI_0000470"	"http://purl.obolibrary.org/obo/OBI_0000470"
Study Assay Technology Type Term Source REF	"OBI"	"OBI"
Study Assay Technology Platform	"micrOTOF-Q"
```
Yours, Steffen

**Comment by Tomnl** [comment]
Hi Steffen,

Thanks for finding this. It seems the "Study" template is not setup correctly to handle this situation. 

I will look into this.

Cheers,
Tom

**Comment by rsalek** [comment]
Thx

**Comment by Tomnl** [comment]
This issue should be fixed now see #28
